react/src/components/atoms/forms/Form/Form.stories.js

```diff
@@ -0,0 +1,147 @@
+import React from 'react';
+
+import { storiesOf } from '@storybook/react';
+import { withInfo } from '@storybook/addon-info';
+import {object, withKnobs, array, text} from '@storybook/addon-knobs/react';
+
+import Form, { FormProvider } from './index';
+import FormInfo from './Form.md';
+import InputNumber from '../InputNumber';
+import InputNumberOptions from '../InputNumber/InputNumber.knobs.options';
+import InputSliderOptions from '../InputSlider/InputSlider.knobs.options';
+import InputSlider from '../InputSlider';
+import InputCurrency from '../InputCurrency';
+import InputCurrencyOptions from '../InputCurrency/InputCurrency.knobs.options';
+
+storiesOf('atoms/forms', module).addDecorator(withKnobs)
+  .add('Form', withInfo(`<div>${FormInfo}</div>`)(() => {
+    delete InputNumberOptions.defaultValue;
+    const inputTextOptionsWithKnobs = Object.assign(...Object.entries(InputNumberOptions).map(([k, v]) => (
+      { [k]: v() })));
+    delete InputSliderOptions.ticks;
+    delete InputSliderOptions.domain;
+    delete InputSliderOptions.labelText;
+    const inputSliderOptionsWithKnobs = Object.assign(...Object.entries(InputSliderOptions).map(([k, v]) => (
+      { [k]: v() })));
+    inputSliderOptionsWithKnobs.labelText = text('InputSlider.labelText', 'Slider (Linked to Input 3)');
+    inputSliderOptionsWithKnobs.ticks = object('InputSlider.ticks', { 0: '0%', 60: 'Minimum requirement', 100: '100%' });
+    const formTicks = object('Form.ticks', { 0: '0%', 60: 'Minimum requirement', 100: '100%' });
+    const ticks = [];
+    const domain = array('Form.domain', [0, 100]).map((x) => Number(x));
```

Comment:
![screen shot 2019-02-07 at 9 41 41 am](https://user-images.githubusercontent.com/5789411/52419849-ea072380-2abe-11e9-936a-c9dcc2d4195e.png)

can you set values to [0,1] so that the value label is representing the percentage correctly?

Reply:
That sounds like a bug on the display value. Slider values will not always be a percentage.